dataplane/internal/engine/engine.go

```diff
@@ -516,7 +545,7 @@ func (e *Engine) actionsFromRoute(ctx context.Context, route *dpb.Route) ([]*fwd
 		}
 	case *dpb.Route_NextHops:
 		var err error
-		actions, err = e.addNextHopList(ctx, hop.NextHops)
+		actions, err = e.addNextHopList(ctx, hop.NextHops, dpb.GroupUpdateMode_GROUP_UPDATE_MODE_UNSPECIFIED)
```

Comment:
Would adding a new enum value be better than UNSPECIFIED? Can use that as default behaviour.

Reply:
changed unspecified to ERROR ON CONFLICT, by making id 0, if unset it's the default bahvior